internal/node/model/models.go

```diff
@@ -46,39 +41,60 @@ const (
 	DefaultBlockStatusSafe      DefaultBlock = "SAFE"
 )
 
+const (
```

Comment:
What does `EpochStatusReceivedLastInput` mean? Is that the same as "epoch closed"? Why not call it that?

Also, I think `EpochStatusClaimComputed` is better than `EpochStatusCalculatedClaim`.

And we could do the same reversion for the other names: "ClaimSubmitted", "ClaimAccepted", ...

Reply:
Changed